src/index.js

```diff
@@ -18,9 +18,20 @@ import 'babel-polyfill';
 import fs from 'fs';
 import path from 'path';
 import vorpal from 'vorpal';
-import { version } from '../package.json';
+import semver from 'semver';
+import { version, engines } from '../package.json';
 import config from './utils/config';
 
+const nodeVersionError = (required, actual) =>
+	chalk.red(`
+Error: Wanted node version ${required} but has ${actual}
+`);
+
+if (!semver.satisfies(process.version, engines.node)) {
+	console.error(nodeVersionError(engines.node, process.version));
+	process.exit(0);
```

Comment:
Do we definitely want to error and exit, or just warn and try to continue?

Reply:
I think exiting make more sense if we are forcing to use some particular version